### PR TITLE
add sql migration file to upgrade process 14.0->15.0

### PR DESCRIPTION
**Pull request by cfoellmann**
sql migration file for 15.0 was not in the install/migration yet

### CODE REVIEW

File: htdocs/install/check.php [hunk 1]
```diff
@@ -411,26 +411,28 @@
...
+			array( 'from' => '5.0.0', 'to' => '6.0.0' ),
+			array( 'from' => '6.0.0', 'to' => '7.0.0' ),
+			array( 'from' => '7.0.0', 'to' => '8.0.0' ),
+			array( 'from' => '8.0.0', 'to' => '9.0.0' ),
+			array( 'from' => '9.0.0', 'to' => '10.0.0' ),
+			array( 'from' => '10.0.0', 'to' => '11.0.0' ),
+			array( 'from' => '11.0.0', 'to' => '12.0.0' ),
+			array( 'from' => '12.0.0', 'to' => '13.0.0' ),
+			array( 'from' => '13.0.0', 'to' => '14.0.0' ),
+			array( 'from' => '14.0.0', 'to' => '15.0.0' ),
```
Comment (ksar-ksar) [inline htdocs/install/check.php:435]:
```suggestion
			array( 'from' => '14.0.0', 'to' => '15.0.0' )
```

Comment (cfoellmann) [reply htdocs/install/check.php:435]: the comma does not hurt and when you add it new lines (array items) do not change the old line

Comment (ksar-ksar) [reply htdocs/install/check.php:435]: Ok

Comment (frederic34) [reply htdocs/install/check.php:435]: https://github.com/Dolibarr/dolibarr/pull/18190

**Comment by cfoellmann** [comment]
Done in #18190

**Pull request closed without merging**